v2/internal/frontend/desktop/windows/screen.go

```diff
@@ -0,0 +1,117 @@
+//go:build windows
+// +build windows
+
+package windows
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/wailsapp/wails/v2/internal/frontend/desktop/windows/winc/w32"
+	"syscall"
+	"unsafe"
+)
+
+func MonitorsEqual(first w32.MONITORINFO, second w32.MONITORINFO) bool {
+	// Checks to make sure all the fields are the same.
+	// A cleaner way would be to check identity of devices. but I couldn't find a way of doing that using the win32 API
+	return first.DwFlags == second.DwFlags &&
+		first.RcMonitor.Top == second.RcMonitor.Top &&
+		first.RcMonitor.Bottom == second.RcMonitor.Bottom &&
+		first.RcMonitor.Right == second.RcMonitor.Right &&
+		first.RcMonitor.Left == second.RcMonitor.Left &&
+		first.RcWork.Top == second.RcWork.Top &&
+		first.RcWork.Bottom == second.RcWork.Bottom &&
+		first.RcWork.Right == second.RcWork.Right &&
+		first.RcWork.Left == second.RcWork.Left &&
+		first.DwFlags == second.DwFlags
```

Comment:
I didn't know if comparing `RcMonitor` only instead of also comparing `RcWork` was enough to guarantee uniqueness of a monitor. Does `RcMonitor` carry enough uniqueness to drop the comparison with `RcWork`

Reply:
Sorry, I meant you have `first.DwFlags == second.DwFlags` twice

Reply:
Ah, I missed that. Yes, I'll remove that.